src/examples/deployment.yml

```diff
@@ -16,25 +16,8 @@ usage:
       steps:
         - checkout
         - kube-orb/install-kubectl
-        - run:
```

Comment:
Thanks for removing this! Could you also remove the `CHANGE_MINIKUBE_NONE_USER` environment variable and change the executor like the following?

```
docker:
      - image: cimg/base:stable
```

Reply:
fixed in https://github.com/CircleCI-Public/kubernetes-orb/pull/26/commits/02a7350ea8c3166c48082f5ae9a2b83d55aa3e47